fe/spark-dpp/src/main/java/com/starrocks/load/loadv2/dpp/Roaring64Map.java

```diff
@@ -1296,7 +1296,7 @@ public void serialize(DataOutput out) throws IOException {
         Codec.encodeVarint64(highToBitmap.size(), out);
 
         for (Map.Entry<Integer, BitmapDataProvider> entry : highToBitmap.entrySet()) {
-            out.writeInt(entry.getKey().intValue());
+            out.writeInt(Integer.reverseBytes(entry.getKey().intValue()));
```

Comment:
could we always call reverseBytes?

Reply:
it seems it's OK in Java
HLL did the process:
```java
            case HLL_DATA_EXPLICIT:
                output.writeByte(type);
                output.writeByte(hashSet.size());
                for (long value : hashSet) {
                    output.writeLong(Long.reverseBytes(value));
                }
                break;
            case HLL_DATA_SPARSE:
```

Reply:
please comment the reason.

Reply:
I add the comment